spec/acceptance/mod_passenger_spec.rb

```diff
@@ -188,10 +171,10 @@ class { 'apache::mod::passenger': }
       expected_one = [%r{Apache processes}, %r{Nginx processes}, %r{Passenger processes}]
       # passenger-memory-stats output on newer Debian/Ubuntu verions do not contain
       # these two lines
-      unless (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') == '14.04') ||
-             (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') == '16.04') ||
-             (fact('operatingsystem') == 'Debian' && fact('operatingsystemmajrelease') == '8') ||
-             (fact('operatingsystem') == 'Debian' && fact('operatingsystemmajrelease') == '9')
+      unless (fact('operatingsystem') == 'Debian' && fact('operatingsystemmajrelease') == '8') ||
```

Comment:
This is essentially all supported distros in the Debian osfamily per metadata.json, right?

Reply:
Is now checking if the os family is debian